uber/automated_emails_server.py

```diff
@@ -319,3 +319,63 @@ def __init__(self, conf):
                                 when=days_before(7, conf.deadline),
                                 sender=c.STAFF_EMAIL,
                                 extra_data={'conf': conf})
+
+
+def notify_admins_of_any_pending_emails():
+    """
+    Generate an email a report which alerts admins that there are emails which are ready to send,
+    but won't because they need approval from an admin.
+
+    This is useful so we don't forget to let certain categories of emails send.
+    """
+    if not c.ENABLE_PENDING_EMAILS_REPORT or not c.PRE_CON or not (c.DEV_BOX or c.SEND_EMAILS):
+        return
+
+    pending_email_categories = get_pending_email_data()
+    if not pending_email_categories:
+        return
+
+    subject = c.EVENT_NAME + ' Pending Emails Report for ' + localized_now().strftime('%Y-%m-%d')
+    body = render('emails/daily_checks/pending_emails.html', {'pending_email_categories': pending_email_categories})
+    send_email(c.STAFF_EMAIL, c.STAFF_EMAIL, subject, body, format='html', model='n/a')
+
+
+DaemonTask(notify_admins_of_any_pending_emails, interval=300,        name="mail placeh")
```

Comment:
Agreed, `mail pending` would be better, or `mail to approve` or something. Additionally, I expect you didn't mean to include a bunch of spaces after that comma. :)